common/council_tasks/00_steward_tasks.txt

```diff
@@ -584,9 +584,8 @@ task_promote_culture = {
 				desc = STEWARD_PROMOTE_CULTURE_COMMUNAL_IDENTITY_BONUS_MODIFIER
```

Comment:
Vrykul have both longships and sea-faring.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/113168530-cfbbd100-9255-11eb-9423-a2e127da6b4c.png)

</details>

Reply:
I thought that was the point, should they only have longships then?

Reply:
> I thought that was the point, should they only have longships then?

If it's the point you prob need to balance it and discover the innovation.